### PR TITLE
Add matching of label placeholders

### PR DESCRIPTION
**Pull request by carsonoid**
This change is [Reviewable](https://reviewable.io/reviews/cesanta/docker_auth/195)

### CODE REVIEW

File: auth_server/authz/acl.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/cesanta/docker_auth/auth_server/authn"
 	"github.com/cesanta/glog"
+	"github.com/schwarmco/go-cartesian-product"
```
Comment (rojer) [inline auth_server/authz/acl.go]: this needs to be vendored

**Comment by rojer** [comment]
this looks reasonable to me, but can you please add unit tests for this? we have `acl_test`, even ;)

**Comment by carsonoid** [comment]
I'll work on the requested changes asap.

File: auth_server/authz/acl.go (hunk 1 above)
Comment (carsonoid) [reply auth_server/authz/acl.go]: I have already added it to the vendor.json. Is there something else that needs to be done to vendor it?

**Comment by carsonoid** [comment]
Ok tests written and PR updated.

**Comment by carsonoid** [comment]
@rojer is there anything else you need from me on this?

**Comment by rojer** [comment]
looks good, thanks!

(sorry, as always simply busy with other stuff)

**Comment by carsonoid** [comment]
Thanks for that @rojer would you be so kind as to cut a new version of the Docker Hub image for me?

**Comment by rojer** [comment]
@carsonoid built and pushed 1.3, currently in `cesanta/docker_auth:1.3`
will promote to `:1` and `:stable` after a few days.

**Comment by carsonoid** [comment]
Perfect! Thank you!

**Pull request merged**